modules/aws-load-balancer-operator-install.adoc

```diff
@@ -0,0 +1,286 @@
+// Module included in the following assemblies:
+//
+// * networking/aws-load-balancer-operator.adoc
+
+:_content-type: PROCEDURE
+[id="aws-load-balancer-operator-install_{context}"]
+= Installing an AWS Load Balancer Operator
+Users that wish to install an AWS Load Balancer Operator (ALBO) can do so as long as the following prerequisites are met:
+
+.Prerequisites
+
+* An existing
+ifndef::openshift-rosa[]
+{product-title}
+endif::openshift-rosa[]
+ifdef::openshift-rosa[]
+{product-title} (ROSA)
+endif::openshift-rosa[]
+cluster with bring-your-own-VPC (BYO-VPC) configuration across multiple availability zones (AZ) installed in STS mode
+* Cluster-admin access
+* Access to modify the VPC and subnets of the created
+ifndef::openshift-rosa[]
+{product-title}
+endif::openshift-rosa[]
+ifdef::openshift-rosa[]
+ROSA
+endif::openshift-rosa[]
+cluster
+// * AWS account credentials
+* Configured
+ifndef::openshift-rosa[]
+{product-title}
+endif::openshift-rosa[]
+ifdef::openshift-rosa[]
+ROSA
+endif::openshift-rosa[]
+CLI (`rosa`)
+* Configured Amazon Web Services (AWS) CLI
+* Configured OpenShift (oc) CLI
+* OpenShift Container Platform (OCP) 4.13 or later
```

Comment:
```suggestion
* {product-title} (OCP) 4.13 or later
```
- Use an attribute for OCP
- Do you need an attribute for the version numbe? I'm not sure here. Just wanted to raise it as something to look at.

Reply:
Need to look into.